cryptoinvestor/views/assets.py

```diff
@@ -2,9 +2,11 @@
 import json
 
 from cryptoinvestor.views import BaseView
-
+from flask import Flask, request
 
 class AssetsListView(BaseView):
+    methods = ['GET']
+    
```

Comment:
There are some whitespaces at the end of the line, please remove it.

I assume you used `vscode` for editing. Then you can turn on whitespace rendering by setting `editor.renderWhitespace` to `all`. And also you can set `files.trimTrailingWhitespace` to `true`, this will automatically delete any trailing whitespace on file save.
Other smart IDEs should also have similar options.